app/models/project.rb

```diff
@@ -542,11 +542,23 @@ def self.find_best!(platform, name, includes = [])
       .compact
       .map(&:downcase)
 
-    visible
+    project = visible
       .lower_platform(platform)
       .where("lower(projects.name) in (?)", names)
       .includes(includes.present? ? includes : nil)
       .first!
+
+    # Track these lookups for a while and see if they actually work. Currently
+    # we're picking the last "repo-url" part of ?go-get=1, which is the VCS url,
+    # but we'd want the first "root-path" part to get actual go module name.
+    if project && platform == "Go"
+      StructuredLog.capture(
+        "GO_MODULES_PROJECT_FIND_NAMES_SUCCESS",
+        { name: name, project_find_names: names, project_id: project.id }
+      )
+    end
```

Comment:
the absence of these logs should prove that the "?go-get=1" code is not doing anything for us